homework/index.js

```diff
@@ -29,18 +29,68 @@
     });
     return elem;
   }
-
+  //get repos
   function main(url) {
-    fetchJSON(url, (err, repositories) => {
+    fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-        return;
+      } else {
+        //createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        const header = createAndAppend('header', root, { class: 'header' });
+        const h1 = createAndAppend('h1', header, { text: 'FooCoding Repos', class: 'h1' });
+        const select = createAndAppend('select', header, { class: 'select' });
+        createAndAppend('option', select, { text: 'Choose your favorite repo' });
+        data.forEach(repo => {
+          const name = repo.name;
+          createAndAppend('option', select, { text: name });
+        });
+        const container = createAndAppend('div', root, { class: 'container' });
+        const repoInfo = createAndAppend('div', container, { class: 'left-div' });
+        const contribs = createAndAppend('div', container, { class: 'right-div' });
+        select.onchange = evt => {
```

Comment:
Instead of using `select.onchange = function` you should use `select.addEventListener('change', function)`. Setting the event-listener like this works in most cases but sometimes breaks and is hard to troubleshoot when it does.